### ISSUE TITLE
Move old shared files to the new HfLA Shared Drive > Food Oasis folder

### ISSUE THREAD

**Issue by fancyham** [hackforla/food-oasis#1087]
### Overview (Nov 21, 2022) 
See latest comments below and #1361 Food Oasis Documentation Assessment for current status.

- [ ] Review Old Google Drive to see if any relevant docs need to be moved over to the New Drive.

### Overview (Nov 2, 2021)
Hack for LA now has an official shared Google Drive, so let's move all our files from the old shared folder to the official new one.  This new drive is set up to prevent accidental deletions, losing files, etc. and will serve as a proper shared drive and archive as volunteers roll on and off.

The new, shared FOLA drive:
https://drive.google.com/drive/u/1/folders/1QCGQH0G5ZnbcLfWj6YU_wQsxezbQlYQd

Why:
This is the safe way to share files which prevents accidental deletions and removals (which happened with the old shared folder)

- In May 2021, a ton of files got yanked out of the old shared Google Drive and returned to the drives of the individual file owners, causing mass confusion.  Still not sure why that happened but that won't happen with the new drive.

If there files were recently returned to your personal Google Drive from the FOLA shared folder, please re-add them to the new FOLA Google Docs folder.  Thank you.

What happened:
- The old shared FOLA folder had permissions issues that apparently caused files to disappear or become unmanageable when volunteers come and go.  (It was also likely a google bug in either the shared folder itself or a UX issue.)
- The new one Google Drive is owned by Hack for LA and any files put in there becomes 'owned' by Hack for LA, preventing accidental deletions or removals.

### Action Items
Please transfer your files into the locations they should be in the new drive.

1. If you have files on the current shared folder, please find them and move them to the new drive.
Old shared folder: 
https://drive.google.com/drive/u/0/folders/1nVn3v_kD9GJ2IO6A84DZkox9h1eqMeRO

New shared FOLA Drive
https://drive.google.com/drive/u/1/folders/1QCGQH0G5ZnbcLfWj6YU_wQsxezbQlYQd

2. If you had files returned to your drive that used to be in the shared FOLA folder,

In Google Drive on the web, select each file, click the (i) button to show details and history.  View the history to see where the file used to reside.  Please put the file in the analogous folder on the new drive.

New shared FOLA Drive
https://drive.google.com/drive/u/1/folders/1QCGQH0G5ZnbcLfWj6YU_wQsxezbQlYQd

**Comment by fancyham** [comment]
If we have more instructions, video, screenshots, please add them here?

Also, please feel free to rewrite issue for clarity, etc.
@entrotech @gigicobos

**Comment by staceyrebekahscott** [comment]
@rylantalerico What was your reason for adding this to the In Progress column? There is no one assigned to it.

**Comment by staceyrebekahscott** [comment]
Closing issue #938 - It pertains to reorganizing the Google Drive. As it originated in April 2021, it is outdated and a new issue should be created to match the organizational needs of the new Drive. Upon completion of this issue, we should revisit the need to develop rules on how and where files should live in the new Drive.

**Comment by staceyrebekahscott** [comment]
@fancyham @entrotech What is the current status of files being moved over from the old drive to the new one? Is there any progress notes or doc that was tracking if all of the files have been moved or how much more there is to do? 

The internship was working on this and has a process built for how to best get this completed, but I'd like to get a sense of where this is currently before we can start thinking about next steps.

**Comment by fancyham** [comment]
I don’t have the current status — perhaps someone or the intern can run a diff to compare the two hierarchies of folders and files?

I do recall that lots of files went missing in early 2021 — files on the shared folder were yanked from the shared folder and returned to the owners, and any one needing to access those files via their links (which stayed the same) had to request access from the original author.  I’m also curious if those yanked files were recovered.

**Comment by staceyrebekahscott** [comment]
Moving this to the Icebox to be reprioritized after the Impact Sprints.

**Comment by staceyrebekahscott** [comment]
See issue #1361 Food Oasis Documentation Assessment for current status of the Food Oasis Google Drive.

**Comment by staceyrebekahscott** [comment]
Text from John D in Slack: 

John Darragh
  [1 day ago](https://hackforla.slack.com/archives/CMER3R1RD/p1668975662477219?thread_ts=1668895431.207689&cid=CMER3R1RD)

FYI, the folders were arranged the way they were to be consistent with the old folder structure until we could move all the old files from the old folder structure to the "new" shared folder structure.

The "old" folder structure was a G-Drive folder that was not controlled by the Hack for LA organization, so each file had an author that owned the file, which was problematic because only the author could perform some operations on the file.  

Sometime around Nov 2021, Hack for LA created the shared folder that we use now. When you create (or upload?) a document there, its ownership is changed to Hack for LA, which is how all our project documentation should work, so ownership can be controlled by an administrator.

The process of moving files from the old folder structure to the new one is extremely painful, as you need to contact the original author of each document and get them to move each file from the old folder structure to the new shared drive in the corresponding folder.   

To make this work, we set up the new shared drive with exactly the same folder structure as the old drive, so the authors could just move the older folders to the corresponding folder in the new shared drive.

The idea was that AFTER all the files had been moved, transferring ownership to Hack for LA, then we could delete the old folder structure entirely and re-arrange the shared drive.

Bonnie personally put a lot of work into contacting the various authors of our old documents and walking them through the process of moving files. I think she had to do it herself because she had the admin permission on the shared drive.


John Darragh
[1 day ago](https://hackforla.slack.com/archives/CMER3R1RD/p1668976456677729?thread_ts=1668895431.207689&cid=CMER3R1RD)

Issue #1087 was in the In Progress column for a long time because it was waiting for Bonnie to contact the remaining authors and walk through moving files, so we could re-organize the new shared drive appropriately.

However, I don't think we would ever be able to contact all of the authors and get all of the files moved even if we put in a lot of effort, so we have probably reached the point of diminishing returns and should abandon the effort.

I'm thinking that maybe the best thing to do is have a few people look through the old folder to find documents that might still be important.  You can make a copy of a document and put the copy in the new shared drive - the problems with this are that you lose all the file history and any links to the original document that may have been referenced elsewhere (like Github issues or wiki pages) will either be broken or point to the original version of the document.

**Comment by staceyrebekahscott** [comment]
My response in Slack: 

Thank you for these details, @John Darragh. I have copied and added them to Issue 1087, which is currently in the Icebox. 

It was my assessment as well, that after 1 year, making the effort to contact former team members to retrieve files is not worth the effort. 

However, it would be worth the effort to go through the old folder and see if there is anything relevant. A copy is better than nothing. 

I'll add this comment, and task to go through the old drive to Issue 1087 (Icebox) and Issue 1361 (In Progress).

**Comment by staceyrebekahscott** [comment]
See task in Overview (Nov, 21 2022) to review old drive for relevant documents.

**Comment by qiqicodes** [comment]
- Closing this issue, the new drive has the most updated documents.
